Library/Formula/camlp5.rb

```diff
@@ -11,12 +11,13 @@ class Camlp5 < Formula
     sha256 "60312c75054d1db597c8afaf2d0563d41e192480531b74b2ad40908e36bbfb8b" => :mountain_lion
   end
 
-  depends_on "ocaml"
+  deprecated_option "strict" => "with-strict"
+  option "with-strict", "Compile in strict mode"
```

Comment:
If we change this, we'll also need to play with [this logic](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/coq.rb#L1-L11).